Makefile

```diff
@@ -16,8 +16,8 @@ test_db:
 	done
 	docker compose exec postgres sh -c 'psql -U postgres -c "drop database if exists tests;" && psql -U postgres -c "create database tests;"'
 
-create_database: .env
```

Comment:
Ironically, I think the Makefile is one place where it's been `create_database` for ages, and has been documented that way.

Can you please switch this Makefile target back to `create_database` so we don't need to change the docs? :smile: